quickwit-search/src/tests.rs

```diff
@@ -310,34 +310,51 @@ async fn test_single_node_filtering() -> anyhow::Result<()> {
         &*test_sandbox.metastore(),
         test_sandbox.storage_uri_resolver(),
     )
-    .await?;
-    assert_eq!(single_node_response.num_hits, 0);
-    assert_eq!(single_node_response.hits.len(), 0);
-
+    .await;
+    assert!(single_node_response.is_err());
+    assert_eq!(
+        single_node_response.err().map(|err| err.to_string()),
+        Some("Invalid query: Field does not exists: 'tag'".to_string())
```

Comment:
Can you fix the code that raises this error:
```Invalid query: field `tag` does not exist.```

Reply:
I believe the error message comes from tantivy's `QueryParserError::FieldDoesNotExist`.